### PR TITLE
Change release body entry to be collapsible

### PR DESCRIPTION
**Pull request by dbast**
A collapsible is specifically useful when there are lots of artefacts with links to be listed.

Split out from https://github.com/crazy-max/ghaction-virustotal/pull/124

Tested here https://github.com/dbast/miniforge/releases/tag/4.12.0-10

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ async function runForReleaseEvent(vt: VirusTotal) {
   core.info(`Release event detected for ${github.context().ref} in this workflow. Preparing to scan assets...`);
 
   const release = await github.getRelease(octokit, github.context().ref.replace('refs/tags/', ''));
-  release.body = release.body.concat(`\n\n🛡 [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal) analysis:`);
```
Comment (crazy-max) [inline src/main.ts:65]: Let's just keep the link as it was before.

File: src/main.ts [hunk 2]
```diff
@@ -78,17 +78,18 @@ async function runForReleaseEvent(vt: VirusTotal) {
...
         await vt.files(await github.downloadReleaseAsset(octokit, asset, path.join(context.tmpDir(), asset.name))).then(upload => {
           outputAnalysis.push(`${asset.name}=${upload.url}`);
           core.info(`${asset.name} successfully uploaded. Check detection analysis at ${upload.url}`);
-          release.body = release.body.concat(`\n* [\`${asset.name}\`](${upload.url})`);
+          release.body = release.body.concat(`\n  * [\`${asset.name}\`](${upload.url})`);
         });
       }
     });
   });
+  release.body = release.body.concat(`\n\ncreated by [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal)</details>\n`);
```
Comment (crazy-max) [inline src/main.ts:92]: I prefer to have the link as it was before. See prev comment.

**Review by crazy-max — CHANGES_REQUESTED** [review]
I think it would be better and might make more sense to have a collapsible section when the number of scanned assets is greater than something like 5.

File: src/main.ts (hunk 1 above)
Comment (dbast) [reply src/main.ts:65]: Not possible as written in the git commit: The collapsible summary text is clickable to expand it and thus links won't
be rendered as such ... Thats why adding it at the end.

File: src/main.ts (hunk 2 above)
Comment (dbast) [reply src/main.ts:92]: See comment above.

**Comment by dbast** [comment]
ok. looking into `if assets.length > 5` ...

**Comment by dbast** [comment]
@crazy-max Updated to make it collapsible when more than 5 artefacts.

File: src/main.ts (hunk 1 above)
Comment (crazy-max) [reply src/main.ts:65]: Ah yes indeed didn't catch that

File: src/main.ts [hunk 3]
```diff
@@ -78,17 +83,20 @@ async function runForReleaseEvent(vt: VirusTotal) {
...
           outputAnalysis.push(`${asset.name}=${upload.url}`);
           core.info(`${asset.name} successfully uploaded. Check detection analysis at ${upload.url}`);
-          release.body = release.body.concat(`\n* [\`${asset.name}\`](${upload.url})`);
+          release.body = release.body.concat(`\n  * [\`${asset.name}\`](${upload.url})`);
         });
       }
     });
   });
+  if (assets.length > 5) {
+    release.body = release.body.concat(`\n\ncreated by [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal)</details>\n`);
```
Comment (crazy-max) [inline src/main.ts:98]: There is an extra newline that doesn't look quite nice:

```
<details>
  <summary>🛡 VirusTotal analysis</summary>

  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)

created by [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal)
</details>
```

image

___

I think it looks better with a `<sub>` like this instead:

```
<details>
  <summary>🛡 VirusTotal analysis</summary>
  
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
<sub>These scans were produced by the [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal)</sub>
</details>
```

```suggestion
    release.body = release.body.concat(`\n<sub>These scans were produced by the [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal)</sub></details>\n`);
```

image

**Review by crazy-max — CHANGES_REQUESTED** [review]

**Comment by dbast** [comment]
Ok. incorporated all proposals. thanks!

**Review by crazy-max — APPROVED** [review]
LGTM thanks!

**Pull request merged**